kubernetes/chart/zulip/values.yaml

```diff
@@ -145,6 +145,8 @@ zulip:
     enabled: true
     accessMode: ReadWriteOnce
     size: 10Gi
+    # -- Set storageClass to use.
+    storageClass:
```

Comment:
What's the thinking on having this set without a specific value? With `existingClaim` below, we have it commented out.

Reply:
This is the same practice as in many charts, if it has no value, it is ignored and using the default.

Reply:
You could also set to an empty string, like: https://github.com/bitnami/charts/blob/6f1e1550bad7dcb0d5cf63e8280baa2d9d7787da/bitnami/gitea/values.yaml#L242 - but the result would be the same. If we add an schema to the chart. it would be best to use the empty string to clearify.